src/pages/index.js

```diff
@@ -6,3 +6,4 @@ export { PlaylistPage } from "./playlist-page/PlaylistPage";
 export { PlaylistDetailPage } from "./playlist-detail-page/PlaylistDetailPage";
 export { LikePage } from "./like-page/LikePage";
 export { WatchLaterPage } from "./watch-later-page/WatchLaterPage";
+export { HistoryPage } from "./history-page/HistoryPage";
```

Comment:
You can export similar files in one file and import them in a single line.

Reply:
sure, noted.